shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -398,11 +407,11 @@ private void setupFlutterEngine() {
       // Defer to the Activity that owns us to provide a FlutterEngine.
       Log.d(TAG, "Deferring to attached Activity to provide a FlutterEngine.");
       FlutterEngineProvider flutterEngineProvider = (FlutterEngineProvider) attachedActivity;
-      flutterEngine = flutterEngineProvider.getFlutterEngine(getContext());
```

Comment:
is this just an API swap?

Reply:
Yeah. I realized in this PR that there were 2 very similar methods called `getFlutterEngine` so I wanted to differentiate.

Reply:
Talked offline, resolving old comments.